VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -49,7 +49,7 @@
                 android:id="@+id/download_in_progress_bar"
                 android:orientation="vertical"
                 style="@style/download_in_progress_bar_style"
-
+                android:background="@color/cyan_1"
```

Comment:
`edx_brand_primary_x_light` is the closest to cyan_1, but it's still noticeably darker:
## cyan_1:
![last_accessed_cyan_1](https://cloud.githubusercontent.com/assets/4244109/8706596/0d03eca6-2b4a-11e5-9fc4-71aff69c7870.png)
## edx_brand_primary_x_light:
![last_accessed_edx_brand_primary_x_light](https://cloud.githubusercontent.com/assets/4244109/8706595/0cfe0656-2b4a-11e5-94cf-854fc63b2af6.png)

Note that this is already used in the "Last Accessed" section, and we're just matching it here. If we're replacing the colors in the app, then we should get a one-to-one mapping, and we can do a global refactoring.